src/HearThis/Utils/NonFatalProblem.cs

```diff
@@ -0,0 +1,138 @@
+﻿using System;
```

Comment:
Breaking the error handling stuff out into a separate library might be okay, but with the current way it's split across two DLLs (SIL.Core and SIL.Windows.Forms), you'd end up with two new DLLs. In practice, I would guess that every one of our apps would end up needing both the original DLL(s) and the new one(s), so I'm not sure it would buy you very much. If this is merely an attempt to try to deal with this one bug, and then the Toast code is getting ripped out, then I guess this approach is okay, but it does seem like a generally useful thing which I could see us using in Glyssen and SayMore at least, so I'd be in favor of moving it into libpalaso. At least it would be a "infrastructure" change that would be a step forward.